packages/edit-site/src/components/sidebar-navigation-screen-navigation-menu/delete-modal.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+	Button,
+	Modal,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+export default function RenameModal( { onClose, onConfirm } ) {
+	return (
+		<Modal title={ __( 'Delete' ) } onRequestClose={ onClose }>
+			<form>
+				<VStack spacing="3">
+					<p>
+						{ __(
+							'Are you sure you wish to delete this Navigation menu?'
```

Comment:
@jasmussen Are you happy with this wording or would you prefer `want` over `wish`?

Reply:
Small preference for want, but not strong at all.